packages/mobile/src/send/TransferConfirmationCard.tsx

```diff
@@ -174,4 +188,4 @@ const style = StyleSheet.create({
   },
 })
 
-export default withNamespaces(Namespaces.sendFlow7)(TransferConfirmationCard)
+export default withNamespaces(Namespaces.sendFlow7)(React.memo(TransferConfirmationCard))
```

Comment:
Is it worth it to memo this?

Reply:
To be honest, I reproduced the way that was exported other component that is a function instead of a class. Should I remove it?